tabs/setup.js

```diff
@@ -137,9 +137,13 @@ TABS.setup.initialize = function (callback) {
                 return;
             }
 
-            configuration_backup(function () {
-                GUI.log(chrome.i18n.getMessage('initialSetupBackupSuccess'));
-            });
+            if (semver.gte(CONFIG.flightControllerVersion, "3.0.0")) {
```

Comment:
The way you are doing it, `configuraton_backup()` is only called for versions >= 3.0. This is not the desired behaviour. Probably the best way to fix this is to extract the call to `configuration_backup()` into a named function of its own, and use that as the callback for `MSP.send_message`, but also directly call it from the `else` clause for versions prior to 3.0.
(see https://github.com/betaflight/betaflight-configurator/blob/master/tabs/configuration.js#L38-L54 for an example).

Re `flightControllerVersion` vs. `apiVersion`: A lot of the configurator code these days is shared between Betaflight and cleanflight. Using `flightControllerVersion` creates additional work when moving code between these two projects, since it will obviously have to be adjusted. For this reason, please use `apiVersion` for all new code. There is a list that allows you to look up what `apiVersion` to use instead of a given `flightControllerVersion`:
https://github.com/cleanflight/cleanflight-configurator/commit/305f827654057618858e10f48a0473fd50d748ca